es/lib/isHexColor.js

```diff
@@ -1,5 +1,5 @@
 import assertString from './util/assertString';
-var hexcolor = /^#?([0-9A-F]{3}|[0-9A-F]{6})$/i;
+var hexcolor = /^#?([0-9A-F]{3}|[0-9A-F]{4}|[0-9A-F]{6}|[0-9A-F]{8})$/i;
```

Comment:
Same here, unrelated.

Reply:
A hex color now can have up to 8 characters, the last 2 digits refers to the opacity of the color. For that the regex edit makes sense and it works for all possible cases. (e.g #22222252)